src/Components/Notifications/Notifications.tsx

```diff
@@ -2,6 +2,13 @@ import { Tab } from "@artsy/palette"
 import { NofiticationsTabs, NofiticationsTabsProps } from "./NotificationsTabs"
 import { NotificationsListQueryRenderer } from "./NotificationsList"
 import { NotificationPaginationType } from "./types"
+import { useEffect, useCallback } from "react"
+import { DateTime } from "luxon"
+import { markNotificationsAsSeen } from "./Mutations/markNotificationsAsSeen"
+import { useSystemContext } from "System"
+import createLogger from "Utils/logger"
+
+const logger = createLogger("MarkNotificationsAsSeen")
```

Comment:
usually we pass the component name to `createLogger`. Something like this`const logger = createLogger("Notifications")`